src/index.html

```diff
@@ -13,6 +13,54 @@
     />
```

Comment:
Keep your attributes correctly formatted. The attributes of the <link> tag should start each on a new line with 2-space indentation related to the tag.